packages/client/hmi-client/src/composables/notificationManager.ts

```diff
@@ -0,0 +1,130 @@
+import { subscribe } from '@/services/ClientEventService';
+import { ClientEvent, ClientEventType, ExtractionStatusUpdate } from '@/types/Types';
+import { NotificationItem } from '@/types/common';
+import { ref, computed } from 'vue';
+import { getDocumentAsset } from '@/services/document-assets';
+import { useToastService } from '@/services/toast';
+import { useProjects } from './project';
+
+let initialized = false;
+
+const { findAsset } = useProjects();
+
+// Items stores the notifications for all projects
+const items = ref<NotificationItem[]>([]);
+
+const toastTitle = {
+	[ClientEventType.ExtractionPdf]: {
+		success: 'PDF Extraction Completed',
+		error: 'PDF Extraction Error'
+	}
+};
+
+const displayToast = (
```

Comment:
Please use the Logger interface for that.
`packages/client/hmi-client/src/utils/logger.ts`

Reply:
Yeap I'll make the change on my next PR